temporal-test-server/src/main/java/io/temporal/internal/testservice/TestWorkflowService.java

```diff
@@ -912,6 +921,125 @@ public void pollWorkflowExecutionUpdate(
     }
   }
 
+  @Override
+  public void executeMultiOperation(
```

Comment:
Do we plan to add explicit test coverage for the multi-ops API as part of this PR?

Reply:
Right; it's only tested implicitly right now. To be honest, until now I didn't see that the test server actually has its own tests. I can add explicit tests there.

Reply:
Added explicit tests here: https://github.com/temporalio/sdk-java/pull/2199/commits/90a6e0ce3513a5d8a52c7f393a2d5d9aa48bdcd2